### PR TITLE
Added Power Of Two in Kotlin

### LINKED ISSUE #6792: Power Of Two in Kotlin
## 🚀 Feature
Given a number to check whether it is a Power of 2 or not

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?
Yes

## Motivation
Since it is not available in the Kotlin directory it motivated me to add one.

## Pitch
It is a good problem to understand the basics of bit manipulations

### PR DESCRIPTION
**Pull request by kiruba-r11**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Closes: #6792

### CODE REVIEW

**Comment by kiruba-r11** [comment]
@ankitaggarwal23 @HarshCasper @bhav09 Please review this

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top
- Use proper code Format 
- Remove extra lines of code and spaces
- Add sample I/O at the end of the Code
- Add Time and Space Complexity at the end of the Code   
- Add new line at the EOF

Add a repl.it link as well

**Comment by kiruba-r11** [comment]
@ankitaggarwal23 I have already added all the changes that you requested

File: Kotlin/BitManipulation/PowerOfTwo.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import java.util.Scanner
+
+/*
+
+A Kotlin Program to find whether a given number is a power of 2 or not.
+
+*/
```
Comment (ankitaggarwal23) [inline Kotlin/BitManipulation/PowerOfTwo.kt:5]: Shift above the header

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by kiruba-r11** [comment]
@ankitaggarwal23 I have done it. Please check it again

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**